api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.proto

```diff
@@ -29,6 +29,7 @@ message AWSLambdaPerRoute {
   // body is used if the body empty, and an empty body will be sent upstream.
   google.protobuf.StringValue empty_body_override = 4;
 
+  // Deprecated. Use transformer_config to specify an AWS Lambda response transformer instead.
```

Comment:
nit: It's not really a functional change, but we could also add the deprecated field option: https://developers.google.com/protocol-buffers/docs/proto#options

Reply:
Added the deprecated field option here: fdf3fa903711b4ad10416c0b7939e1d225295033